p5/sketch/renderer3d.py

```diff
@@ -33,13 +34,22 @@
 from ..core.shape import PShape
 
 from ..pmath.matrix import translation_matrix
-from .openglrenderer import OpenGLRenderer, get_render_primitives
-from .shaders3d import src_default, src_fbuffer
+from .openglrenderer import OpenGLRenderer, get_render_primitives, to_3x3
+from .shaders3d import src_default, src_fbuffer, src_normal
+
+
+class Shader(Enum):
+	BASIC = 'BASIC'
+	NORMAL = 'NORMAL'
+	BLINN_PHONG = 'BLINN_PHONG'
```

Comment:
Should this be in separate class for reflection model?

Reply:
I think (glossy) reflection can be covered in one blinn_phong shader. Could you elaborate on what may be separated?

Reply:
I was thinking of different classes for materials and reflection models. In the future, if they have same structure, then its fine to combine them in the same `Shader` class.